### DISCUSSION TITLE
Is there any prepackaged Hyprland distro?

### DISCUSSION THREAD

**Discussion by PSGitHubUser1** [mylinuxforwork/dotfiles#184] [Q&A]
_The title_

 I want hyprland with the themes like in Archcraft. But its paid...

**Comment by Bammede** [comment]
[wayblue](https://github.com/wayblueorg/wayblue)
[wayblue hardened](https://github.com/secureblue/secureblue)

**Comment by PSGitHubUser1** [comment]
Any other that has a good looking terminal? Or any way to customise it?
